x-pack/plugins/aiops/public/embeddables/change_point_chart/embeddable_chart_component_wrapper.tsx

```diff
@@ -101,10 +101,6 @@ export const ChartGridEmbeddableWrapper: FC<ChangePointDetectionProps> = ({
     10000
   );
 
-  useEffect(() => {
```

Comment:
It was giving an error, as `onLoading` is not defined for chart component:
![image](https://github.com/user-attachments/assets/adaf48c0-2042-499c-991d-6c4f122dc24a)